README.md

```diff
@@ -53,9 +58,19 @@ Note: The samples require Zivid SDK v2 (minor version 2.2 or newer).
 
 2. [**Download Zivid Sample Data**][zivid-sample-data-url].
 
-3. Launch Visual Studio 2017.
 
-4. Open and run one of the samples.
+Launch the Command Prompt by pressing *Win + R* keys on the keyboard, then type cmd and press Enter.
+
+Navigate to a location where you want to clone the repository, then run the following command:
+
+```bash
+git clone https://github.com/zivid/zivid-csharp-samples
+```
+
+Open *ZividNETSamples.sln* in Visual Studio, build it and run it. If you are uncertain about doing this, check out our [**tutorials for configuring and building C# Samples**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/534708242).
```

Comment:
Is this link still valid, or should it point to the new kb now?

Reply:
No, corrected with the new KB